modules/pam_unix/support.c

```diff
@@ -122,10 +122,10 @@ int _make_remark(pam_handle_t * pamh, unsigned int ctrl,
  * set the control flags for the UNIX module.
  */
 
-int _set_ctrl(pam_handle_t *pamh, int flags, int *remember, int *rounds,
-	      int *pass_min_len, int argc, const char **argv)
+int _set_ctrl(pam_handle_t *pamh, unsigned long long flags, int *remember,
```

Comment:
This has to return unsigned long long and the flags parameter actually should stay as int.

Reply:
Fixed and changed the return type.